product/views.py

```diff
@@ -4,9 +4,36 @@
 from django.http  import JsonResponse
 from django.db    import transaction, IntegrityError
 
-from .models      import MatchingReview, Review
-from order.models import Order
-from utils         import login_decorator
+from user.models    import User
+from order.models   import Order
+from product.models import ProductLike, Product, MatchingReview, Review
+from utils          import login_decorator
+
+class ProductLikeView(View):
+    @login_decorator
+    def get(self, request):
+        
+        likes = ProductLike.objects.filter(user_id = request.user.id)
```

Comment:
```suggestion
        likes = ProductLike.objects.filter(user_id = request.user.id)
```
request에 들어가있는 user객체 활용해서 좋아요한 상품을 불러오시는군요 ! good good !
ProductLike클래스는 user 객체를 참조 하고 있으므로, user 인스턴스에서 역참조 관계를 이용하여 productlike 속성에 접근하실 수 있습니다 :)

ProductLike 클래스를 새로 부르는 방법 외에, user에서 타고 들어가서 가져오는 방법으로 한 번 응용해보세요!
`likes = user.____blahblah`의 형태로 접근해보시기 바랍니다.

Reply:
네! 역참조로 수행하겠습니다!